Sources/_CryptoExtras/AES/AES_CBC.swift

```diff
@@ -84,6 +102,18 @@ extension AES {
         ///   - iv: The initialization vector.
         /// - Returns: The decrypted message.
         public static func decrypt<Ciphertext: DataProtocol>(_ ciphertext: Ciphertext, using key: SymmetricKey, iv: AES._CBC.IV) throws -> Data {
+            try self.decrypt(ciphertext, using: key, iv: iv, noPadding: false)
+        }
+        
+        /// Decrypts data using AES-CBC.
+        ///
+        /// - Parameters:
+        ///   - ciphertext: The encrypted ciphertext.
+        ///   - key: A decryption key.
+        ///   - iv: The initialization vector.
+        ///   - noPadding: If this is set to `true`, padding won't be removed.
+        /// - Returns: The decrypted message.
+        public static func decrypt<Ciphertext: DataProtocol>(_ ciphertext: Ciphertext, using key: SymmetricKey, iv: AES._CBC.IV, noPadding: Bool = false) throws -> Data {
```

Comment:
Let's not default the value here.

Reply:
Yeah, I don't know how that made it into the commit :), changed it now